deploy/README.md

```diff
@@ -0,0 +1,8 @@
+# Deploying the search client to a custom Wordpress page template
+
+1. Run `npm run build` to build this project.
+2. Figure out which Wordpress theme you're using and navigate to the directory for that theme under wp\_content.
```

Comment:
It took me awhile to figure out, but I think the README has escaped `_` for some reason. Could you confirm so I can remove the `\`s?

Reply:
I thought that _'s needed to be escaped in markdown files, but they don't need to be escaped. I removed the \'s.